### PR TITLE
auth: completely disable the packet when cache-ttl=0

### PR DESCRIPTION
**Pull request by mind04**
This was inspired by #7802 but is more in line with the current query cache behaviour

closes #7802

### Checklist
I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [x] compiled this code
- [x] tested this code
- [x] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: docs/settings.rst [hunk 1]
```diff
@@ -180,7 +180,8 @@ Also AXFR a zone from a master with a lower serial.
 -  Integer
 -  Default: 20
 
-Seconds to store packets in the :ref:`packet-cache`.
+Seconds to store packets in the :ref:`packet-cache`. A value of 0 will disable
```
Comment (Habbie) [inline docs/settings.rst]: Please use one line per sentence. We don't do that everywhere yet, but we should :)

**Comment by Habbie** [comment]
ci/circleci: test-auth-regress-odbc-mssql failure can be ignored.

File: pdns/packethandler.cc [hunk 2]
```diff
@@ -1546,8 +1546,7 @@ DNSPacket *PacketHandler::doQuestion(DNSPacket *p)
     if(doSigs)
       addRRSigs(d_dk, B, authSet, r->getRRS());
       
-    r->wrapup(); // needed for inserting in cache
```
Comment (Habbie) [inline pdns/packethandler.cc]: was this comment wrong?

Comment (mind04) [reply pdns/packethandler.cc]: There is no need to do it here getString() will warpup when necessary https://github.com/powerdns/pdns/blob/master/pdns/dnspacket.cc#L63

**Pull request merged**